src/components/EditRegistrationAccordion.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import { Collapse } from "antd";
+import CreateRegistration from "./CreateRegistration";
+import SelectHandle from "./SelectHandle";
+import { HexString } from "../utilities/types";
+import { DSNPUserURI } from "@dsnp/sdk/dist/types/core/identifiers";
+import * as registry from "@dsnp/sdk/core/contracts/registry";
+const { Panel } = Collapse;
+
+interface EditRegistrationAccordionProps {
+  isCreatingRegistration: boolean;
+  walletAddress: HexString;
+  onIdResolved: (uri: DSNPUserURI) => void;
+  registrations: registry.Registration[];
+}
+
+const EditRegistrationAccordion = ({
+  isCreatingRegistration,
+  walletAddress,
+  onIdResolved,
+  registrations,
+}: EditRegistrationAccordionProps): JSX.Element => {
+  return (
+    <Collapse
+      bordered={false}
+      defaultActiveKey={isCreatingRegistration ? ["0"] : ["1"]}
```

Comment:
maybe consider the case, if logged in and userID is set, then have both panels closed?